### PR TITLE
Update config docs (cleanup, packit_instances, issue_repository)

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
* Remove outdated example of Packit config.
* Document `packit_instances` and usage of staging app.
* Document `issue_repository` config option

### CODE REVIEW

File: content/docs/configuration.md [hunk 1]
```diff
@@ -277,6 +277,15 @@ notifications:
 
 You can enable the commenting by setting `successful_build` to `true`.
 
+#### issue_repository
+
+Use this key to be notified about errors of the downstream jobs (Koji build and Bodhi update).
+This repository can be any GitHub/GitLab/Pagure repository
+where issues are enabled and Packit has an identity on that instance.
+(Let us know if you need some other instance to be supported.)
```
Comment (majamassarini) [inline content/docs/configuration.md:285]:
```suggestion
This repository can be any GitHub/GitLab/Pagure instance
where issues are enabled and Packit has an identity.
(Let us know if you need some other instances to be supported.)
```

File: content/docs/configuration.md [hunk 2]
```diff
@@ -320,14 +329,24 @@ instead of the location defined in the spec-file.
...
+
+Just be aware that:
+* You still need to install [the staging GitHub application](https://github.com/apps/packit-as-a-service-stg).
+* Staging application has fewer resources and can be a bit slower in responses.
+* If you use both instances, you will have twice the number of commit statuses.
+* Picking just one instance might be better for downstream jobs since
+  both instances will work with the production instances of Fedora systems.
+
+Please, let us know when something looks weird or does not work with the staging app.
+By doing that, you are helping us be sure that we don't break your use-case.
```
Comment (majamassarini) [inline content/docs/configuration.md:349]:
```suggestion
By doing that, you are helping us to be sure that we don't break your use-case.
```

**Review by majamassarini — COMMENTED** [review]
My english is bad and I hope this suggestions sounds good to you, otherwise feel free to skip them :)

File: content/docs/configuration.md (hunk 1 above)
Comment (lachmanfrantisek) [reply content/docs/configuration.md:285]: I might have mixed those two config options together, but here, we are really talking about repositories. As I am looking at that, I'll make it more clear and mention that it needs to be a URL.

Comment (lachmanfrantisek) [reply content/docs/configuration.md]: Can you please take another look if it is clear now?

**Review by majamassarini — APPROVED** [review]

**Comment by lachmanfrantisek** [comment]
Thanks, Maja!

**Pull request merged**